framework/include/utils/MooseTypes.h

```diff
@@ -452,20 +452,6 @@ using GenericVariableGradient = typename Moose::GenericType<VariableGradient, is
 template <bool is_ad>
 using GenericVariableSecond = typename Moose::GenericType<VariableSecond, is_ad>;
 
-#define declareADValidParams(ADObjectType)                                                         \
```

Comment:
Pgh was using that one I think

This is effectively instant deprecation. I think we will be fine but hard to know, not all apps are in our test suite though.